src/http_server.cc

```diff
@@ -75,10 +75,10 @@ HTTPServer::Start()
     evhtp_enable_flag(htp_, EVHTP_FLAG_ENABLE_NODELAY);
     evhtp_set_gencb(htp_, HTTPServer::Dispatch, this);
     evhtp_use_threads_wexit(htp_, NULL, NULL, thread_cnt_, NULL);
-    if (evhtp_bind_socket(htp_, "0.0.0.0", port_, 1024) != 0) {
+    if (evhtp_bind_socket(htp_, http_address_.c_str(), port_, 1024) != 0) {
       return TRITONSERVER_ErrorNew(
           TRITONSERVER_ERROR_UNAVAILABLE,
-          (std::string("Port '0.0.0.0:") + std::to_string(port_) +
+          (std::string("Port '") + http_address_ + ":" + std::to_string(port_) +
```

Comment:
Fix error message here. The port is just `port_`. A socket is a combination of port and IP address.